### PR TITLE
[medInria-4.0] Remove old hardware shading

### PR DESCRIPTION
**Pull request by Florent2305**
Replace PR #613
refer [medInria-3.1.x] Remove old hardware shading #612

### CODE REVIEW

File: src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.cpp [hunk 1]
```diff
@@ -196,6 +195,8 @@ void medVtkFibersDataInteractorPrivate::setROI (medAbstractData *data)
 
     roiManager->SetInput (converter->GetOutput());
     roiManager->SetDirectionMatrix (matrix2);
+
+    // manager->GetROILimiter()->Update();
```
Comment (mathildemerle) [inline src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.cpp:199]: to remove

File: src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.h [hunk 2]
```diff
@@ -224,3 +223,5 @@ private slots:
 private:
     medVtkFibersDataInteractorPrivate *d;
 };
+
```
Comment (mathildemerle) [inline src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.h:226]: you can remove one of the empty lines

File: src/plugins/legacy/medVtkFibersData/manager/vtkFibersManager.cxx [hunk 3]
```diff
@@ -18,6 +18,7 @@
 #include <vtkCommand.h>
 #include <vtkBoxWidget.h>
 #include <vtkPolyData.h>
+#include <vtkPolyDataMapper.h>
```
Comment (mathildemerle) [inline src/plugins/legacy/medVtkFibersData/manager/vtkFibersManager.cxx:21]: is it needed in this file?

Comment (Florent2305) [reply src/plugins/legacy/medVtkFibersData/manager/vtkFibersManager.cxx:21]: Yes it is

File: src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.h (hunk 2 above)
Comment (Florent2305) [reply src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.h:226]: ok

File: src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.cpp (hunk 1 above)
Comment (Florent2305) [reply src/plugins/legacy/medVtkFibersData/interactors/medVtkFibersDataInteractor.cpp:199]: ok

**Pull request merged**